utils/preprocessing.py

```diff
@@ -68,6 +69,9 @@ def preprocess_co2_emissions(df: pd.DataFrame) -> pd.DataFrame:
     # To work with tonnes of co2 it is necessary a conversion factor of 3.664
     df['co2'] = df['co2'] / 3.664
     df.rename(columns={'date': 'date', 'co2': 'co2_emissions'}, inplace=True)
+    df.to_pickle("./co2_emission")
```

Comment:
Can you save this pickle file in a folder in tmp? So the repo doesnt grow big? You can use:
path_to_results = SDKConfig().get_output_dir("processed_dataframes_pickle_files") for example.

Reply:
Done Done